Add `Vocaloid Mapping Contest 2 & New Featured Artist: iroha(sasaki)` news post and associated wiki article (#10849)
ppy/osu-wiki: news/2024/2024-01-28-vocaloid-mapping-contest-2.md

---
layout: post
title: "Vocaloid Mapping Contest 2 & New Featured Artist: iroha(sasaki)"
date: 2024-01-28 12:00:00 +0000
---

VOCALOID lovers rejoice! The Vocaloid Mapping Contest returns bigger and better than ever, featuring support for every game mode, an improved format, a bit of necromancy, and incredible music from our brand-new Featured Artist: **iroha(sasaki)**!

![](/wiki/Contests/VMC/2/img/banner.jpg)

> Yep, you heard right. **iroha(sasaki)**, the artist behind the legendary VOCALOID classic *Meltdown* featured in *many* classic osu! maps, is now a Featured Artist!

Reviewer: something like this maybe :

```suggestion
Yep, you heard right. **iroha(sasaki)**, the artist behind the legendary VOCALOID classic *Meltdown* featured in *many* classic osu! maps, is now a Featured Artist (and the 400th one to be added to the listing, no less!)
```

*Meltdown* has already earned its place in osu!'s history with several popular Ranked maps, including [Diasios](https://osu.ppy.sh/users/25999)'s [early map of *Meltdown*](https://osu.ppy.sh/beatmapsets/9070#osu/36814) from 2009, [ertyukjh010](https://osu.ppy.sh/users/220939)'s [map and unique storyboard](https://osu.ppy.sh/beatmapsets/45672#osu/142590) from 2012, and even more recently [ohm002](https://osu.ppy.sh/users/4468239)'s [modern marathon](https://osu.ppy.sh/beatmapsets/1743736#osu/3564761) from 2022!

Regardless of how popular *Meltdown* may be, it's far from **iroha(sasaki)**'s only song — we're happy to announce that **11** songs are part of [**iroha(sasaki)**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/399)!

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/artists/399/release_showcase.mp4" type="video/mp4" preload="none">
    </video>
</div>

***BUT WAIT!!!*** This is more than just a Featured Artist announcement!

## Vocaloid Mapping Contest 2

> The [**Vocaloid Mapping Contest** (***VMC***)](/wiki/Contests/VMC) returns, and just like last time, it's all about spotlighting hidden VOCALOID gems!

Reviewer 2: include the contest listing link somewhere here for ease of access

Author: put in the "big changes" section

More precise details regarding the stream schedule etc. will be announced in the [VAT Discord server](https://discord.com/invite/EruvDHaReY) and updated on this news post, once they are finalised. A video of the stream will be made available afterward for those who can't make it!

\**Screening will only occur in modes with a large number of entries.*

### Prizes

What would VOCALOID be without its mascots? For a contest featuring **iroha(sasaki)** songs sung by Kagamine *Rin* and judged by Natsume*Rin*, the first-place osu! winner will receive a *custom VMC Winner Kagamine Rin plushie* handcrafted by none other than [Noffy](https://osu.ppy.sh/users/1541323)!

| Placing | Prize(s) |  |
| :-: | :-- | :-- |
| 1st place | Custom *VMC 2 Winner* VOCALOID plushie\*, 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC 2 Winner* profile badge | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |
| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |  |

> \**Prize only for participants of the osu! game mode, due to it's one-of-a-kind nature.*

Reviewer 3: ```suggestion
\**Prize only for participants of the osu! game mode, due to its one-of-a-kind nature.*
```

This contest is created by and for vocal synth fans to promote music, artists, and mappers. **All mappers who participate will receive constructive feedback from experienced mappers in the community.** That's arguably a more valuable prize in its own right!

### Songs

As proven by this contest, great things can happen when we promote and support the artists who make osu! great.

> Below we have selected 5 of iroha(sakaki)'s incredible works to you to pick from that were appropriate for contest mapping, and we're confident that something will inspire you!

Reviewer 2: ```suggestion
Below we have selected 5 of **iroha(sakaki)**'s incredible works to you to pick from that were appropriate for contest mapping, and we're confident that something will inspire you!
```
new artist is always bold in these articles

#### iroha(sasaki) - The Cuckoo Bird's Beautiful Youth (5:18 / ??? BPM)

**Vocaloid Featured**: VY1 & Kagamine Rin

**Lyricist**: π

[Download the beatmap template here!](LINK)

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

#### iroha(sasaki) - Shirley!! (4:59 / ??? BPM)

**Vocaloid Featured**: Kagamine Rin

**Lyricist**: π

[Download the beatmap template here!](LINK)

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

#### iroha(sasaki) - Devolution (4:26 / ??? BPM)

**Vocaloid Featured**: Kagamine Rin

**Lyricist**: kuma(alfred)

[Download the beatmap template here!](LINK)

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

#### iroha(sasaki) - 14 (Miku ver.) (4:55 / ??? BPM)

**Vocaloid Featured**: Hatsune Miku

**Lyricist**: π

[Download the beatmap template here!](LINK)

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

#### iroha(sasaki) - sigh (4:08 / ??? BPM)

> **Vocaloid Featured**: Kagamine Rin 

Reviewer 4: ```suggestion
**Vocaloid Featured**: Kagamine Rin
```

**Lyricist**: iroha(sasaki)

[Download the beatmap template here!](LINK)

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

> ### Rules

Reviewer: not gonna lie i don't feel like the technical tidbits fit well compared to the rest of the article tbh, especially considering how long the news post already is

i feel like it would be \*much\* better if these two sections are to be removed altogether and be replaced by sth like this

```suggestion
For more information regarding the rules of the contest and how to submit your map, feel free to check out the [VMC 2 wiki](/wiki/Contests/VMC/2).
```

Author: not opposed to it if others think it important

Reviewer 3: all contest news posts have rules in them, so i don't see why removing here is necessary

> - **Submissions must use one of the [selected songs](#songs).**

Reviewer 2: mention if cut versions are allowed (so many people will ask with this song length)

Author: done

| **Download** | [Beatmap template (4:26 / 190 BPM)](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20Devolution.osz) |

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20Devolution.mp3" type="audio/mpeg">
    </audio>
</div>

| Song | iroha(sasaki) - 14 (Miku ver.) |
| :-- | :-- |
| **Vocaloid** | Hatsune Miku |
| **Lyricist** | π |
| **Download** | [Beatmap template (4:55 / 128 BPM)](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.osz) |

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.mp3" type="audio/mpeg">
    </audio>
</div>

| Song | iroha(sasaki) - sigh |
| :-- | :-- |
| **Vocaloid** | Kagamine Rin |
| **Lyricist** | iroha(sasaki) |
| **Download** | [Beatmap template (4:08 / 148 BPM)](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.osz) |

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.mp3" type="audio/mpeg">
    </audio>
</div>

### Rules

> - **Submissions must use one of the [selected songs](#songs).** Cut versions of the songs are not allowed.

Reviewer 3: ```suggestion
- **Submissions must use one of the [songs from the selection below](#songs).** Cut versions are not allowed.
```

- **Your submission must have only one difficulty.**

### Submission

Once your beatmap is ready, head to the relevant contest listing to submit your map:

- [osu! contest listing](https://osu.ppy.sh/community/contests/199)
- [osu!taiko contest listing](https://osu.ppy.sh/community/contests/200)
- [osu!catch contest listing](https://osu.ppy.sh/community/contests/201)
- [osu!mania contest listing](https://osu.ppy.sh/community/contests/202)

After the submission period closes, judging will begin. Find out more details about the judges and the judging criteria on the [contest's wiki page](/wiki/Contests/VMC/2).

---

And that should be everything you need to know to get started!

For the most up-to-date info about the contest, keep the [**Vocaloid Mapping Contest** wiki page](/wiki/Contests/VMC/2) bookmarked. Feel free to also join the [VAT (Vocaloid Appreciation Team) Discord server](https://discord.com/invite/EruvDHaReY) to receive updates on the VMC (as well as future events), map feedback, info on collaborative VOCALOID mapping projects, and more!

We look forward to seeing your maps!

> —-White

Reviewer 5: Isn't pishifat responsible for FA outreach? Maybe add his name on that line if he did that?

Reviewer 3: the author line is usually for the people who wrote the post, i.e. you in this case
